src/ui/components/auth/SignInContent.tsx

```diff
@@ -0,0 +1,145 @@
+/* Copyright (c) 2022, VRAI Labs and/or its affiliates. All rights reserved.
```

Comment:
All the copies need to change (leaving this comment as a reminder so we dont forget)